packages/tools/src/tools/WindowLevelTool.ts

```diff
@@ -7,10 +7,9 @@ import {
   StackViewport,
   utilities,
   cache,
+  Types,
```

Comment:
This was intentioanl import. When we can we should import type with `import type `